core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.html

```diff
@@ -78,4 +78,31 @@
 # once a day on the 1st and 15th of every month except December
 H H 1,15 1-11 *
 </pre>
+<p>
+  <b>Timezones specification</b>. <br/>
+  Starting from 1.615 Jenkins supports the specification 
+  of different timezones in this field. It allows to natively configure all
+  periodic operations.
+  
+  <ul>
+    <li>Timezone can be specified in the first line, which should contain the &quot;TZ=$(timezoneID)&quot; string</li>
+    <li>The plugin supports all timezones returned by 
+      <a href="https://docs.oracle.com/javase/7/docs/api/java/util/TimeZone.html#getAvailableIDs()">
+        java.util.TimeZone::getAvailableIDs()
+      </a> on the Jenkins server JVM
```

Comment:
Agreed, I'll provide a link to the Wiki page. Wikipedia should be OK for such purpose